ruff_lsp/server.py

```diff
@@ -1063,17 +1086,14 @@ async def resolve_code_action(params: CodeAction) -> CodeAction:
 
     settings = _get_settings_by_document(document.path)
 
-    if settings["organizeImports"] and params.kind in (
-        CodeActionKind.SourceOrganizeImports,
-        f"{CodeActionKind.SourceOrganizeImports.value}.ruff",
+    if (
+        settings["organizeImports"]
+        and params.kind == CodeActionKind.SourceOrganizeImports
     ):
         # Generate the "Ruff: Organize Imports" edit
         params.edit = await _fix_document_impl(document, only=["I001", "I002"])
 
-    elif settings["fixAll"] and params.kind in (
-        CodeActionKind.SourceFixAll,
-        f"{CodeActionKind.SourceFixAll.value}.ruff",
```

Comment:
I'm pretty sure we only ever pass the full action here? This is for deferred actions.